rpc/api.go

```diff
@@ -62,6 +62,8 @@ func (api *EthereumApi) GetRequestReply(req *RpcRequest, reply *interface{}) err
 		*reply = newHexData(api.xeth().Coinbase())
 	case "eth_mining":
 		*reply = api.xeth().IsMining()
+	case "eth_hashrate":
+		*reply = api.xeth().HashRate()
```

Comment:
I don't see anything on the spec page for this, but it probably needs to be wrapped in a call to newHexNum() so that the output is consistent with other number formats.